### PR TITLE
Expose app version number at /api

### PR DESCRIPTION
**Pull request by tomwilkie**
I'm thinking we might want to expose a bunch more stuff at /api, such as what probe addresses we're connected to, what version those probes are etc.

### CODE REVIEW

**Comment by davkal** [comment]
LGTM

Not this displaying this for now.

**Comment by tomwilkie** [comment]
Yes yes, baby steps...

**Comment by tomwilkie** [comment]
This is for #145

**Comment by peterbourgon** [comment]
What's the concrete use case? I want to avoid adding things to the API because we might want to have them in the future, maybe.

**Comment by tomwilkie** [comment]
This is to show the app version in the UI, for when people file bug reports.

**Comment by tomwilkie** [comment]
@davkal PTAL

File: app/router.go [hunk 1]
```diff
@@ -35,6 +36,12 @@ func captureTopology(rep Reporter, f func(Reporter, topologyView, http.ResponseW
 	}
 }
 
+func apiHandler(w http.ResponseWriter, r *http.Request) {
+	respondWith(w, http.StatusOK, map[string]string{
+		"version": version,
+	})
+}
```
Comment (peterbourgon) [inline app/router.go]: Please make an APISomething type for this :)

**Comment by peterbourgon** [comment]
Also I'd propose the following patch:

```diff
diff --git a/Makefile b/Makefile
index c2245b2..c171d4f 100644
--- a/Makefile
+++ b/Makefile
@@ -11,7 +11,7 @@ SCOPE_IMAGE=$(DOCKERHUB_USER)/scope
 SCOPE_EXPORT=scope.tar
 SCOPE_UI_BUILD_EXPORT=scope_ui_build.tar
 SCOPE_UI_BUILD_IMAGE=$(DOCKERHUB_USER)/scope-ui-build
-SCOPE_VERSION=$(shell git rev-parse HEAD)
+SCOPE_VERSION=$(shell git rev-parse --short HEAD)

 all: $(SCOPE_EXPORT)
```

**Comment by tomwilkie** [comment]
Good suggestions.  I'll hold off for @davkal as there are a bunch of UI changes too.

**Comment by davkal** [comment]
Let's move this to the right and make it a bit paler:

```diff
diff --git a/client/app/styles/main.less b/client/app/styles/main.less
index 4f9cfa4..6a06eb6 100644
--- a/client/app/styles/main.less
+++ b/client/app/styles/main.less
@@ -58,11 +58,11 @@ body {
 
 .footer {
   position: absolute;
-  bottom: 32px;
-  padding-left: 64px;
-  width: 100%;
-  height: 1em;
+  bottom: 16px;
+  right: 48px;
   z-index: 20;
+  color: @text-tertiary-color;
+  font-size: 85%;
 }
 
 .logo {
```

**Comment by davkal** [comment]
Yeah, rest looks GTM

**Pull request merged**